apps/vast/pyvast_threatbus/message_mapping.py

```diff
@@ -77,7 +78,17 @@ def indicator_to_vast_query(indicator: Indicator) -> Union[str, None]:
     if vast_type == "ip" or vast_type == "ipv6":
         return ioc_value
     if vast_type == "url":
-        return f'"{ioc_value}" in net.uri'
+        if match(r"[A-Za-z0-9]+://", ioc_value):
+            return f'"{ioc_value}" == net.uri'
+        else:
+            protocols = [
+                "",
+                "http://",
+                "tcp://",
+            ]
```

Comment:
This list is not complete. Why only use this particular subset? I'd expect to either see a rather exhaustive list or a list that can be configured by the user, but not a hard-coded very limited list.

Reply:
Did you see the discussion in Slack?

Reply:
Yes. But just because this incomplete patch is applied somewhere else does not that we should implement it like that in the upstream codebase.

Reply:
Per further discussion, this PR is on hold until we decide how to properly handle URIs in our stack.